cookbooks/aws-parallelcluster-platform/resources/spack_install.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright:: 2013-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+provides :spack
+unified_mode true
+
+default_action :install_spack
+
+action :install_spack do
+  spacksh_path = '/etc/profile.d/spack.sh'
```

Comment:
this should probably be `spack_sh_path` or better `spack_script_path` to be more clear